+nirs/+bids/data2JSON.m

```diff
@@ -5,13 +5,13 @@ function data2JSON(data,filename,task)
 
 snirf=nirs.util.validateSNIRF(data);
 
-fprintf(fid,'\t"TaskName": %s\n',task);
-fprintf(fid,'\t"Version": %s\n',snirf.formatVersion);
-fprintf(fid,'\t"MeasurementDate": %s\n',snirf.nirs.metaDataTags.MeasurementDate);
-fprintf(fid,'\t"MeasurementTime": %s\n',snirf.nirs.metaDataTags.MeasurementTime);
-fprintf(fid,'\t"SNIRF_createDate": %s\n',snirf.nirs.metaDataTags.SNIRF_createDate);
-fprintf(fid,'\t"SNIRF_createTime": %s\n',snirf.nirs.metaDataTags.SNIRF_createTime);
-fprintf(fid,'\t"filedescription": %s\n',snirf.nirs.metaDataTags.filedescription);
+fprintf(fid,'\t"TaskName": "%s",\n',task);
+fprintf(fid,'\t"Version": %s,\n',snirf.formatVersion);
+fprintf(fid,'\t"MeasurementDate": "%s",\n',snirf.nirs.metaDataTags.MeasurementDate);
+fprintf(fid,'\t"MeasurementTime": "%s",\n',snirf.nirs.metaDataTags.MeasurementTime);
+fprintf(fid,'\t"SNIRF_createDate": "%s",\n',snirf.nirs.metaDataTags.SNIRF_createDate);
+fprintf(fid,'\t"SNIRF_createTime": "%s",\n',snirf.nirs.metaDataTags.SNIRF_createTime);
+fprintf(fid,'\t"filedescription": "%s"\n',snirf.nirs.metaDataTags.filedescription);
 fprintf(fid,'}');
```

Comment:
This is also fairly insiduous because the json file will not actually read unless these quotes are present, good work

Reply:
thank you :)